spark/src/test/java/org/apache/iceberg/actions/TestRewriteDataFilesAction.java

```diff
@@ -377,4 +378,95 @@ private void writeDF(Dataset<Row> df) {
         .mode("append")
         .save(tableLocation);
   }
+
+  @Test
+  public void testRewriteToOutputPartitionSpec() {
+    PartitionSpec spec = PartitionSpec.builderFor(SCHEMA)
+            .identity("c1")
+            .build();
```

Comment:
Nit: continuing indentation in this method is off. It should be 2 indents / 4 spaces from the start of the previous line.